master/pom.xml

```diff
@@ -2244,7 +2244,7 @@
         <configuration>
           <systemPropertyVariables>
             <!-- disable lombok during tests -->
-	    <lombok.disable>true</lombok.disable>
+	        <lombok.disable>true</lombok.disable>
```

Comment:
tabs vs. spaces here

Reply:
Sakai is tabs or spaces?

Reply:
Depends on the file.... :(

Rule of thumb is to use whatever is most prevalent in the file you're working on, or the most prevalent in the surrounding blocks if it's all mixed up.

Reply:
Have you ever think about creating a .editorconfig file in root?
https://editorconfig.org/

Reply:
Requested Changes done